### PR TITLE
SickRage v4.0.9

### PR DESCRIPTION
**Pull request by petemcw**
This is a new formula for SickRage which is a branched project from Sickbeard, which is already in Homebrew. I built out a system this weekend and figured I'd issue a pull request in case anyone else would be interested.

### CODE REVIEW

**Comment by mistydemeo** [comment]
Can you squash this down to one commit, please? It looks like you've merged branches several times, so extra commits have been pulled in.

**Comment by petemcw** [comment]
@mistydemeo I'm sorry it took me so long to get back, I totally missed your comments. I've squashed the commit down. Let me know if you'd like me to make any other changes.

File: Library/Formula/sickrage.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
+require "formula"
+
+class Sickrage < Formula
+  homepage "http://www.sickrage.tv/"
+  head "https://github.com/SiCKRAGETV/SickRage.git"
+  url "https://github.com/SiCKRAGETV/SickRage/archive/v4.0.9.tar.gz"
+  sha1 "77cbf0b7e5acbc7eaca4211c923d883db383fd71"
+
+  depends_on :python
```
Comment (mikemcquaid) [inline Library/Formula/sickrage.rb]: Should be `depends_on :python if MacOS.version <= :snow_leopard`

File: Library/Formula/sickrage.rb [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      system "python", *Language::Python.setup_install_args(libexec)
+    end
+
+    # create SickRage executible
+    (bin/"sickrage").write(startup_script)
+  end
+
+  def caveats
+    "SickRage defaults to port 8081."
+  end
```
Comment (mikemcquaid) [inline Library/Formula/sickrage.rb]: Is this Homebrew specific?

File: Library/Formula/sickrage.rb [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    # install Sickrage contents into `libexec`
+    libexec.install Dir["*"]
+
+    # install Cheetah into `libexec`
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    resource("Cheetah").stage do
+      system "python", *Language::Python.setup_install_args(libexec)
+    end
+
+    # create SickRage executible
```
Comment (mikemcquaid) [inline Library/Formula/sickrage.rb]: Can remove all the comments in `install` as they are fairly self-explanatory, thanks.

**Comment by petemcw** [comment]
I tried to clean this up a little bit more and updated the version to 4.0.17. @mistydemeo or @mikemcquaid can you check it out again?

File: Library/Formula/sickrage.rb [hunk 4]
```diff
@@ -0,0 +1,62 @@
+class Sickrage < Formula
+  homepage "http://www.sickrage.tv/"
+  head "https://github.com/SiCKRAGETV/SickRage.git"
+  url "https://github.com/SiCKRAGETV/SickRage/archive/v4.0.17.tar.gz"
+  version "4.0.17"
```
Comment (mikemcquaid) [inline Library/Formula/sickrage.rb]: Can remove this.

File: Library/Formula/sickrage.rb (hunk 2 above)
Comment (mikemcquaid) [inline Library/Formula/sickrage.rb]: Remove this.

File: Library/Formula/sickrage.rb [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+  def startup_script; <<-EOS.undent
+    #!/bin/bash
+    export PYTHONPATH="#{libexec}/vendor/lib/python2.7/site-packages:$PYTHONPATH"
+    python "#{libexec}/SickBeard.py"\
+      "--pidfile=#{var}/run/sickrage.pid"\
+      "--datadir=#{etc}/sickrage"\
+      "$@"
+    EOS
+  end
```
Comment (mikemcquaid) [inline Library/Formula/sickrage.rb]: Inline this method.

File: Library/Formula/sickrage.rb [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+end
```
Comment (mikemcquaid) [inline Library/Formula/sickrage.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by DomT4** [comment]
Please address Mike's comments here, Thanks!

**Comment by DomT4** [comment]
Closing due to no response. Will welcome a new PR with the issues highlighted addressed. Thanks for your submission to Homebrew!

**Pull request closed without merging**